code/modules/jobs/job_types/roguetown/church/priest.dm

```diff
@@ -72,12 +72,14 @@ GLOBAL_LIST_EMPTY(heretical_players)
 		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/swimming, 1, TRUE)
+		H.change_stat("strength", 2)
+		H.change_stat("constitution", 1)
+		H.change_stat("endurance", 3) //good at casting lots of miracles without tiring, divinely fortified stamina
+		H.change_stat("intelligence", 3) 
+		H.change_stat("speed", 2) // ducking and weaving in their 5 unarmed combat and 5 wrestling martial arts prowess
```

Comment:
I don't really think the priest should be getting speed stat increases, to be quite honest.

Reply:
Sure would.